src/System.Management.Automation/help/HelpCommands.cs

```diff
@@ -615,22 +619,37 @@ private void LaunchOnlineHelp(Uri uriToLaunch)
                     "https");
             }
 
+            // we use this test hook is to avoid actually calling out to another process
+            if (InternalTestHooks.BypassOnlineHelpRetrieval)
+            {
+                this.WriteObject(string.Format(CultureInfo.InvariantCulture, HelpDisplayStrings.OnlineHelpUri, uriToLaunch.OriginalString));
+                return;
+            }
+
             Exception exception = null;
             try
             {
                 this.WriteVerbose(string.Format(CultureInfo.InvariantCulture, HelpDisplayStrings.OnlineHelpUri, uriToLaunch.OriginalString));
                 System.Diagnostics.Process browserProcess = new System.Diagnostics.Process();
-
 #if UNIX
                 browserProcess.StartInfo.FileName = Platform.IsLinux ? "xdg-open" : /* OS X */ "open";
                 browserProcess.StartInfo.Arguments = uriToLaunch.OriginalString;
-                browserProcess.Start();
 #elif CORECLR
-                throw new PlatformNotSupportedException();
+                // On CoreCLR, ProcessStartInfo.UseShellExecute doesn't exist.
```

Comment:
we don't use `ProcessStartInfo.UseShellExecute` directly near this code. Not very clear from the comment what's the need of UseShellExecute.